cockatrice/src/settingscache.cpp

```diff
@@ -89,6 +91,12 @@ void SettingsCache::setCardInfoViewMode(const int _viewMode) {
     settings->setValue("cards/cardinfoviewmode", cardInfoViewMode);
 }
 
+void SettingsCache::setHighlightWords(const QString &_highlightWords) {
+    // Words are seperated by a comma and you can not use spaces in words
+    highlightWords = _highlightWords.simplified().replace(" ", "").split(",");
```

Comment:
You should stick with one convention. I dont mind which. Either always user X or always use Y.